portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFolderLocalServiceImpl.java

```diff
@@ -853,86 +854,44 @@ public DLFolder updateFolder(
 
 			// File entry types
 
-			DLFolder dlFolder = null;
+			DLFolder dlFolder = dlFolderLocalService.getDLFolder(folderId);
 
 			Set<Long> originalFileEntryTypeIds = new HashSet<>();
 
 			if (folderId > DLFolderConstants.DEFAULT_PARENT_FOLDER_ID) {
 				originalFileEntryTypeIds = getFileEntryTypeIds(
 					dlFolderPersistence.getDLFileEntryTypes(folderId));
 
-				dlFolder = dlFolderLocalService.updateFolderAndFileEntryTypes(
-					serviceContext.getUserId(), folderId, parentFolderId, name,
-					description, defaultFileEntryTypeId, fileEntryTypeIds,
-					restrictionType, serviceContext);
-
-				if (!ExportImportThreadLocal.isImportInProcess()) {
-					_dlFileEntryTypeLocalService.cascadeFileEntryTypes(
-						serviceContext.getUserId(), dlFolder);
+				if (GetterUtil.getBoolean(
```

Comment:
I thing that we are breaking api for the customer that are using local api. Since, the default value is false and for that reason they can't update folder and file entry types.

Reply:
oh, yes I see the default value to true, I got confused, sorry for the noise.